setup.py

```diff
@@ -36,7 +36,7 @@
                     'click',
                     'Jinja2',
                     ],
-    tests_require=['pytest'],
+    tests_require=['pytest < 5'],
```

Comment:
```suggestion
    tests_require=['pytest < 5;python_version<"3.5"', 'pytest'],
```

This migth work.

Reply:
With "pytest < 5":

```
# python2.7 setup.py test --addopts -vv
...
running pytest
Searching for pytest<5
Reading https://pypi.org/simple/pytest/
Downloading https://files.pythonhosted.org/packages/8c/6f/4db6124788e9b281d94693e21e5a40d00f417763dcda10edc8a70655cb92/pytest-4.6.4-py2.py3-none-any.whl#sha256=6aa9bc2f6f6504d7949e9df2a756739ca06e58ffda19b5e53c725f7b03fb4aae
Best match: pytest 4.6.4
Processing pytest-4.6.4-py2.py3-none-any.whl
Installing pytest-4.6.4-py2.py3-none-any.whl to /home/gordon/git/pyp2rpm/.eggs
writing requirements to /home/gordon/git/pyp2rpm/.eggs/pytest-4.6.4-py2.7.egg/EGG-INFO/requires.txt

```

With ['pytest < 5;python_version<"3.5"', 'pytest']:

```
# python2.7 setup.py test --addopts -vv
...
running pytest
Searching for pytest
Reading https://pypi.org/simple/pytest/
Downloading https://files.pythonhosted.org/packages/60/23/de5604e58f4eba7a90f70486c8d4ece25f1a404bae29683903ffd2aea425/pytest-5.0.1.tar.gz#sha256=6ef6d06de77ce2961156013e9dff62f1b2688aa04d0dc244299fe7d67e09370d
Best match: pytest 5.0.1
...
pkg_resources.VersionConflict: (pytest 5.0.1 (/home/gordon/git/pyp2rpm/.eggs/pytest-5.0.1-py2.7.egg), Requirement.parse('pytest<5; python_version < "3.5"'))
```

Reply:
Oh, but adding a platform dependency to the second list item fixes it.  Thanks for the tip.